HerPublicWebsite/Controllers/StaticPagesController.cs

```diff
@@ -36,4 +41,11 @@ public IActionResult SessionExpired()
     {
         return View("SessionExpired");
     }
+    
+    [HttpGet("/digital-assistance")]
```

Comment:
Although the page URI is currently `/digital-assistance`, as this matches conventions of this particular codebase, it does not match the original ask of `/digitalassistance`, we will confirm with the client whether this is ok

Reply:
We should check. My main concern is that we probably want GBIS and HUG2 to match at the very least!